### PR TITLE
DPC 1586: Build new static site layouts

### PR DESCRIPTION
**Pull request by Sun-Mountain**
### Fixes [DPC-1586](https://jira.cms.gov/browse/DPC-1586)

Apply new design and language for landing/home page, new pages for DPC Data, About the Pilot, and Updates.

### Proposed Changes

- Add DPC Data, About the Pilot, and Updates pages
- Update landing page and language

### Change Details

- V2 of Documentation will be released at a later date when we fully deploy the implementer application along with updated documentation that will be linked to V1 documents

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### Acceptance Validation

Screen Shot 2021-07-20 at 10 08 19 AM

Screen Shot 2021-07-20 at 10 13 32 AM

Screen Shot 2021-07-20 at 10 13 43 AM

Screen Shot 2021-07-20 at 10 13 55 AM

Screen Shot 2021-07-20 at 10 14 03 AM

### CODE REVIEW

**Review by megmcf — APPROVED** [review]
Reviewed staging site for updated DPC static site - matches layouts approved by H-team and product

File: _includes/_banner-title.html [hunk 1]
```diff
@@ -1,5 +1,25 @@
...
+          {{ page.button }}<br />
+          (coming soon)
+        </div>
+      </div>
+    {% elsif page.id == "docsV2" %}
+      <div class="banner-btn__container">
+        <a href={{ site.sbx_sign_in }} target="_blank" rel=noopener class="banner-btn">
+          {{ page.button }}
+        </a>
+      </div>
```
Comment (dhgreene) [inline _includes/_banner-title.html:16]: Why does this need a special case?

File: _layouts/page-sidenav.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                <ul>
+                  {% for subitem in item.subnav %}
+                    <li class="sidenav__item">
+                      <a href="#{{ subitem.id }}" class="ds-u-display--block sidenav__link">{{subitem.title}}</a>
+                    </li>
+                  {% endfor %}
+                </ul>
+              {% endif %}
+            </li>
+          {% endfor %}
```
Comment (dhgreene) [inline _layouts/page-sidenav.html:31]: This is the same logic repeated for specific pages. It also doesn't seem that it's temporary (e.g. until releasing the user guide). How can we DRY it up? Can we set up a hash of `page.id` => nav_items?

File: _sass/components/_site-blocks.scss [hunk 3]
```diff
@@ -62,4 +122,156 @@
...
+  background: url("/assets/images/hexagon-bg.svg");
+  padding: $spacer-3 0 $spacer-7 0;
+
+  h2 {
+    text-align: center;
+  }
+}
+
+#hero {
+  // background: url("/assets/images/dpc-header.svg") no-repeat center;
```
Comment (dhgreene) [inline _sass/components/_site-blocks.scss:206]: Can we delete this now?

**Pull request merged**